src/styles/base/_override.typography.scss

```diff
@@ -89,7 +98,7 @@
 
 .ds-text--lead,
 .ds-text-body--lg {
-  font-family: "Rubik";
+  font-family: Rubik;
```

Comment:
Can we use a variable here instead of the font itself?

Reply:
This is complete.